easy-ui-react/src/Menu/Menu.test.tsx

```diff
@@ -141,7 +141,7 @@ describe("<Menu />", () => {
       getMenu({
         menuProps: { defaultOpen: true },
         items: [
-          <Menu.Section key="s-1" aria-label="Section 1">
```

Comment:
**suggestion**: it might be a good idea to add a unit test to check that a title gets rendered